apps/stats-mainnet-e2e/cypress.json

```diff
@@ -0,0 +1,12 @@
+{
+  "fileServerFolder": ".",
```

Comment:
follow up (non-blocking): let's create a ticket to add some cypress tests for this

Reply:
#109